From 24a68f77594bd4b974953cb727117953ff24e3b9 Mon Sep 17 00:00:00 2001
From: Martin Abente Lahaye <tch@sugarlabs.org>
Date: Wed, 30 Oct 2019 10:09:22 -0300
Subject: [PATCH] Don't override existing document-open icon

Remove this old override on a existing icon. Otherwise
the document-open icon is shown as actvity-start.

Looking at b8a1f426, it is clear that this override was
added before we provided our own version of this icon.
---
 icons/scalable/actions/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/icons/scalable/actions/Makefile.am b/icons/scalable/actions/Makefile.am
index 94fd6f0..729c86e 100755
--- a/icons/scalable/actions/Makefile.am
+++ b/icons/scalable/actions/Makefile.am
@@ -134,7 +134,6 @@ EXTRA_DIST = $(icon_DATA)
 # install aliases for icons, and the use the icon-naming-utils to install
 # further aliases for compatibility with gtk stock icon names.
 install-data-local: install-iconDATA
-	ln -sf activity-start.svg $(DESTDIR)$(icondir)/document-open.svg
 	ln -sf activity-stop.svg $(DESTDIR)$(icondir)/application-exit.svg
 	ln -sf dialog-ok.svg $(DESTDIR)$(icondir)/dialog-apply.svg
 	ln -sf dialog-ok.svg $(DESTDIR)$(icondir)/gtk-ok.svg
-- 
2.19.1

